### PR TITLE
docker-compose: basic dc file is added

### PR DESCRIPTION
**Pull request by rkhozinov**
Source was taken from oficial
redmine's dockerhub

https://hub.docker.com/_/redmine/

### CODE REVIEW

**Comment by rkhozinov** [comment]
fix #14

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+version: '2'
+
```
Comment (svd01) [inline docker-compose.yml:2]: no "+"

Comment (rkhozinov) [reply docker-compose.yml:2]: ok

Comment (svd01) [reply docker-compose.yml:2]: lets do it

**Review by svd01 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (rkhozinov) [reply docker-compose.yml:2]: no, dude. '+' sign is related to patch specific syntax

Comment (svd01) [reply docker-compose.yml:2]: ok. my mistake

**Review by svd01 — APPROVED** [review]

**Comment by rkhozinov** [comment]
close #5

**Pull request merged**